pom.xml

```diff
@@ -54,6 +54,24 @@
         <version.modelmapper>3.2.0</version.modelmapper>
         <version.assertj-core>3.25.3</version.assertj-core>
         <version.h2>2.2.224</version.h2>
+        <test.exclusions>
+            **/testgenerator/**,
+            **/yaciindexer/**,
+            **/org/openapitools/**,
+            **/model/**,
+            **/entity/**,
+            **/rosetta/config/**,
+            **/rosetta/common/enumeration/NetworkId.java,
+            **/rosetta/common/enumeration/OperationTypeStatus.java,
+            **/rosetta/common/enumeration/RewardType.java,
+            **/rosetta/common/enumeration/SyncStateType.java,
+            **/rosetta/common/enumeration/ScriptPurposetype.java,
+            **/rosetta/api/mempool/service/MempoolServiceImpl.java,
```

Comment:
Could you add a comment here, that says something like "excluded for now, will be integrated when Mempool is implemented completely" just as an example :)